src/build.ts

```diff
@@ -36,24 +37,18 @@ export class Builder {
         this.targetDir = targetDir;
     }
 
-    static init(): Builder {
-        const targetDir = process.cwd();
-        try {
-            const workerDir = path.join(targetDir, "worker");
+    static createWorkerDir(workerTargetDir: string) {
+        const workerDir = path.join(workerTargetDir, "worker");
```

Comment:
I find suspicious that we have a constructor with parameter `targetDir` and then a static method with parameter `workerTargetDir`. Shouldn't it be an instance method and reuse the constructor parameter?